deps/github.com/arangodb/go-driver/context.go

```diff
@@ -52,11 +52,12 @@ const (
 	keyIsSystem                 ContextKey = "arangodb-isSystem"
 	keyIgnoreRevs               ContextKey = "arangodb-ignoreRevs"
 	keyEnforceReplicationFactor ContextKey = "arangodb-enforceReplicationFactor"
-	keyConfigured               ContextKey = "arangodb-configured"
+	keyConfigured  	             ContextKey = "arangodb-configured"
```

Comment:
1. The go-driver is vendored in this repo. Never change it here, without a PR in the go-driver repo first.
2. VST is controlled by the type of `Connection`, no need to add anything to the context for that.